plugins/scaffolder/src/api.ts

```diff
@@ -109,6 +113,23 @@ export class ScaffolderClient implements ScaffolderApi {
       .filter(c => options.allowedHosts.includes(c.host));
   }
 
+  async getIntegration(options: { type: string }) {
```

Comment:
Use the integrations API directly instead, don't see the need to add this to the scaffolder API